src/components/ButtonPanel.js

```diff
@@ -0,0 +1,42 @@
+/*
+ eslint-disable no-unused-vars
+*/
+
+import React from 'react';
+import Button from './Button';
+
+const ButtonPanel = props => (
+  <div>
+    <div>
+      <Button btnName="AC" />
+      <Button btnName="+/-" />
+      <Button btnName="%" />
+      <Button btnName="/" />
+    </div>
+    <div>
+      <Button btnName="7" />
+      <Button btnName="8" />
+      <Button btnName="9" />
+      <Button btnName="X" />
+    </div>
+    <div>
+      <Button btnName="4" />
+      <Button btnName="5" />
+      <Button btnName="6" />
+      <Button btnName="-" />
+    </div>
+    <div>
+      <Button btnName="1" />
+      <Button btnName="2" />
+      <Button btnName="3" />
+      <Button btnName="+" />
+    </div>
+    <div>
+      <Button btnName="0" />
+      <Button btnName="." />
+      <Button btnName="=" />
+    </div>
+  </div>
+);
+
```

Comment:
- [ ] There is a better way of doing this. Try putting all button types in an array and looping over that array to render the buttons. Thats way, you reduce on code duplication

Reply:
Not a good idea, because it causes issues with styling the groups of Buttons.